test/unit/classification_test.rb

```diff
@@ -314,8 +302,9 @@ def setup
                           :value => "345",
                           :use_puppet_default => true
     end
-    enc = classification.enc
-    refute enc['base'].has_key?(key.key)
+
+    enc = @host.puppetclass_parameters
+    refute enc['base'].key?(key.key)
```

Comment:
Saw a few of this `has_key` to `key` changes, you may want to do it everywhere via rubocop (in another PR)